odd-platform-api/src/test/resources/application-integration-test.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    url: jdbc:tc:postgresql:13.2-alpine:///test-db
```

Comment:
you should also define 
```driver-class-name: org.testcontainers.jdbc.ContainerDatabaseDriver```

Reply:
From the doc it will be detected by provided url. I suppose we don't need to define it manually.
https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#data.sql.jdbc
![image](https://user-images.githubusercontent.com/32108047/145673192-0a0c5b92-f45a-40ae-ad29-06273e78d443.png)

Also form the code
![image](https://user-images.githubusercontent.com/32108047/145673324-716192c2-9c36-4b78-8dbf-52b9f124b143.png)